build.override.targets

```diff
@@ -55,4 +57,16 @@
     <Message Text="$([System.IO.File]::ReadAllText($(WcfSelfHostCleanupLog)))" Condition="Exists('$(WcfSelfHostCleanupLog)')" Importance="High" />
     <Message Text="-------------- WCF clean end --------------" Importance="High" />
   </Target>
+
+  <!-- This target will copy a master version of project.json that uses last released version of non-WCF dependencies to the scenario test dirs. -->
+  <!-- These non-WCF dependencies "may" explicitly reference a pre-release version but should only do so to pull in needed fixes. -->
+  <Target Name="WcfCopyProjectJson" Condition="'$(UseStableDependencies)'=='true'">
```

Comment:
+1 to remove the condition.  And the copy inside will (as you do already) unconditionally copy from $(ScenariosDependencyJson) which will have been set outside this target.  There should not be any decisions inside this target about where to copy from or whether it is using stable or latest.